test/assert.js

```diff
@@ -1,5 +1,6 @@
 describe('assert', function () {
   var assert = chai.assert;
+  var expect = chai.expect;
```

Comment:
Let's not add `expect` to the `assert` interface tests. It should be possible to test errors without `expect` by using the same style that's used elsewhere in this file.